README.md

```diff
@@ -255,6 +260,9 @@ You can also use this tool to get a list of syscalls needed by a program:
 
 ## Uninstalling
 
+Note: To remove desktop integration for a given user, run `sudo firecfg
+--clean` as that user.
```

Comment:
IIRC we never delete in `~/.local/share/applications`. `--clean` only removes the symlinks. Therefore `sudo`-vs-root does not matter.

Reply:
> IIRC we never delete in `~/.local/share/applications`. `--clean` only removes
> the symlinks. Therefore `sudo`-vs-root does not matter.

Good catch; removed.